composer.json

```diff
@@ -369,9 +370,6 @@
                 "Issue #3215817: Presentational attributes such as 'border', 'align', or 'bgcolor' are used. CSS should be used for styling instead.": "https://git.drupalcode.org/project/video_embed_field/-/merge_requests/3.diff",
                 "Add support for Ckeditor 5, see https://www.drupal.org/project/video_embed_field/issues/3311063#comment-15227318": "https://www.drupal.org/files/issues/2023-09-12/3311063-35.patch"
             },
```

Comment:
we have duplication of this patch https://github.com/YCloudYUSA/yusaopeny/blob/a63d1c181ce52689225221e0e78cbb5960d283ae/composer.json#L372